HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 History
 -------
 
+7.0.23 (2017-09-14)
```

Comment:
We don't increment versions in the same commits/PR as code changes. Also, can you rebase your commit message to something more informative and a single commit instead of two?

Reply:
Roger, I'll fix it up this afternoon!